### ISSUE TITLE
show attached images when printing to PDF

### ISSUE THREAD

**Issue by leonardolins2021** [yllen/pdf#67]
Is there a way to show images placed in the body of the ticket's PDF to appear in the printout?

**Comment by laurixia** [comment]
True, when exporting tickets to PDF, follow-ups containing images are not displayed.

**Comment by laurixia** [comment]
solved with this solution https://github.com/yllen/pdf/commit/2473f4bea007ae6725381b985e66a1b21f820d7c

![image](https://github.com/yllen/pdf/assets/26741776/3220b629-c58f-4d73-bc3b-7fdeedcfcb10)
